readyplayerme/Unity-Avatar-Loader
# [3639140773] Add scripts from core

## [3639140773](https://ready-player-me.monday.com/boards/2563815861/pulses/3639140773)

## Description

-  Move avatar loader specific scripts from core to avatar loader.

## Changes

#### Added

- All data classes
- Editor settings classes

#### Updated

- Adapted AvatarlObjectLoader according to decoupling of OperationExecutor from CustomException.

## How to Test

-  Best way to test is to check [this](https://github.com/readyplayerme/Unity-SDK/pull/16) PR in Unity-SDK repo 



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
